.github/workflows/dependabot_lint_format.yml

```diff
@@ -66,3 +66,9 @@ jobs:
         with:
           commit_message: Format & lint pnpm-lock.yaml
           file_pattern: "./pnpm-lock.yaml"
+
+      - name: "Run lint and jest if there are changes"
+        if: steps.auto-commit-action.outputs.changes_detected == 'true'
+        run: |
+          pnpm lint-check
+          pnpm test
```

Comment:
1. We're you going to add `pnpm build` as well?
2. What's the difference running them like this vs running them individually?
```yaml
      - name: "Lint if there are changes"
        if: steps.auto-commit-action.outputs.changes_detected == 'true'
        run: pnpm lint-check

      - name: "Test if there are changes"
        if: steps.auto-commit-action.outputs.changes_detected == 'true'
        run: pnpm test
```